torch/_higher_order_ops/_map.py

```diff
@@ -0,0 +1,324 @@
+from functools import partial
```

Comment:
This file was moved from functorch/experimental/_map.py. It just doesn't look like a move due to not completely deleting functorch/experimental/_map.py